auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -139,6 +139,12 @@
         <!-- Status -->
         <template #item-slot-Status="{ item }">
           <span>{{ status(item) }}</span>
+          <!-- Future Effective Icon for PAID status -->
+          <template v-if="status(item) === BusinessState.PAID">
+            <v-icon class="future-effective-icon">
+              mdi-information-outline
+            </v-icon>
+          </template>
```

Comment:
1. A filing may be in Paid (or Approved) status temporarily if the Filer is stuck. I think you also need to check the future-effective flag.
2. A tooltip should be attached to this icon. Andy has prepared a short tooltip that includes the FE date.

Reply:
Just to confirm:
- Paid and Future Effective: show Future Effective and the tooltip that includes the FE date
- Paid and not FE (the Filer is stuck): show Paid.

Reply:
Correct!

Reply:
Updated!
- Paid and Future Effective (approved or not-approved with no other status): show Future Effective and the tooltip that includes the FE date
- Paid and not FE (the Filer is stuck): show Paid | Pending.
- Approved: show Approved if it's not a FE

Reply:
Perfect, thanks!